test/library/standard/Types/is-default-initializable-arrays.future

```diff
@@ -1,2 +0,0 @@
-bug: cannot create array of non-default-initializable record
-#15233
```

Comment:
@dlongnecke-cray / @vasslitvinov : The mention of this issue number and lack of other futures storing that number makes me wonder whether issue #15233 can be closed now?

Reply:
It would make sense to try out various combinations of arrays, tuples, non-default-initializable records, and non-nilable classes before closing #15233.